### ISSUE TITLE
Regression in v1.2.0: @if (...) and (...)

### ISSUE THREAD

**Issue by CatoTH** [scssphp/scssphp#165]
Hi!

Taking the following code:
```php
$scss = new \ScssPhp\ScssPhp\Compiler();
$scss->setFormatter(\ScssPhp\ScssPhp\Formatter\Compact::class);
echo $scss->compile('$test: "1";
$test2: "2";
@if ($test == "1") and ($test2 == "2") {
  .debug {color:red;}
}');
```

Version 1.1.1 outputted the following:
```css
.debug { color:red; }
```

Version 1.2.0 outputs the following:
```css
@if ($test == "1") and ($test2 == "2") { .debug { color:red; }
```

This is an issue when compiling for example some old bootstrap versions (e.g. https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_grid-framework.scss#L34)

**Comment by Cerdic** [comment]
faulty commit is f7e3e745df03b1271646e7e12614bdc800f6e851

**Comment by Cerdic** [comment]
This is fixed bug f634760
